packages/chains/src/polygon/plugins.ts

```diff
@@ -4,30 +4,37 @@ const deployedPlugins: DeployedPlugins = {
   "0x6578e774120F6010315784C69C634bF3946AFb0c": {
     market: "0x9b5D86F4e7A45f4b458A2B673B4A3b43D15428A7",
     name: "Beefy agEUR-jEUR Vault",
+    docsUrl: "https://app.beefy.com/",
   },
   "0x74bA0D32B7430a2aad36e48B7aAD57bf233bDDa6": {
     market: "0xCC7eab2605972128752396241e46C281e0405a27",
     name: "Beefy jEUR-PAR Vault",
+    docsUrl: "https://app.beefy.com/",
```

Comment:
Ideally, we have more specific links in this case: https://app.beefy.com/vault/jarvis-2eurp since we can access them. might need some work here, but should be doable

Reply:
Done